src/Web3Provider/BrowserWeb3Provider.ts

```diff
@@ -29,28 +32,42 @@ export class BrowserWeb3Provider implements IWeb3Provider {
   public async connectWallet() {
     const { default: Torus } = await import('@toruslabs/torus-embed')
     const providerOptions: IProviderOptions = {
-      torus: this.walletSetting?.providers?.torus
-        ? {
-            package: Torus, // required
-            display: this.walletSetting?.providers?.torus?.display,
-            options: {
-              config: this.walletSetting?.providers?.torus?.options,
-            },
-          }
-        : { package: Torus },
+      walletconnect: {
+        package: WalletConnectProvider,
+        options: {
+          ...this.walletSetting?.providers?.walletconnect?.options,
+        },
+      },
+      torus: {
+        package: Torus,
+        display: this.walletSetting?.providers?.torus?.display,
+        options: {
+          config: this.walletSetting?.providers?.torus?.options,
+        },
+      },
     }
 
     this.web3Modal = new Web3Modal({
-      providerOptions,
       cacheProvider:
         this.walletSetting?.selectWalletModal?.cacheProvider ?? false,
       theme: this.walletSetting?.selectWalletModal?.theme ?? 'light',
+      providerOptions: getSelectedWalletProvider(
```

Comment:
Why do we need this place?

Reply:
@tomonari-t -san. This code allows deleting providers in `ProviderOptions` that are not selected in `WalletSetting providers`

Reply:
@longlh1997 
I think we do not have to delete it.

Reply:
@tomonari-t -san. I created them so that in case the user only uses `torus` or `walletconnect` when initializing the MintSDK, the other provider will not be shown in the login method on the website

Reply:
@longlh1997 
Oh I got it.

Reply:
[L34](https://github.com/KyuzanInc/mint-sdk-js/pull/166/files#diff-0371edcfe82aee0567c20b68b2c0e4d555f65a170880ec4dbd88d53f0e94f275R34) to apply only the wallets set by the user would be simpler. What do you think?

Reply:
What do you think of the following example?
```typescript
const providerOptions: IProviderOptions = {
    torus: {
      package: Torus,
   }
}
this.walletSetting?.providers?.torus && providerOptions['torus'] = {
  package: Torus,
  display: this.walletSetting?.providers?.torus?.display,
  options: {
    config: this.walletSetting?.providers?.torus?.options,
  },
}
this.walletSetting?.providers?.torus && providerOptions['walletconnect'] =  {
    package: WalletConnectProvider,
    options: {
      ...this.walletSetting?.providers?.walletconnect?.options,
    },
  }

```

Reply:
Thank you @tomonari-t -san. I will update them to follow your example

Reply:
@tomonari-t -san. I have fixed it. Please check it. Thanks